<?xml version="1.0"?>
<launch>

  <arg name="exp"/>
  <arg name="task"/>
  <!--arg name="cond"/-->
  <arg name="sim" default="false"/>

  <!-- manipulation remote monitor application -->
  <node pkg="srs_user_tests" name="bb_overlap_node" type="bb_overlap_node" output="screen">

    <!-- value defines what will be considered as sucessful bb placement (overlap) -->
    <param name="bb/success_val" type="double" value="0.85"/>
    
    <param name="gr/success_val" type="double" value="0.05"/>

    <!-- minimal duration of sucessful overla -->
    <param name="bb/success_min_dur" type="double" value="2.0"/>

    <param name="gr/success_min_dur" type="double" value="2.0"/>

    <param name="publish_debug_markers" type="bool" value="false"/>

    <!-- load predefined position of BB -->
    <rosparam if="$(arg sim)" file="$(find srs_user_tests)/ros/config/$(arg exp)/$(arg task)/bb_position_sim.yaml" command="load" />

    <rosparam unless="$(arg sim)" file="$(find srs_user_tests)/ros/config/$(arg exp)/$(arg task)/bb_position.yaml" command="load" />

  </node>



</launch>
